cards/src/main/resources/cards/custom/group0/other_death_knight/ungoro/spell_bat_swarm.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "Bat Swarm",
+  "baseManaCost": 7,
+  "type": "SPELL",
+  "heroClass": "SPIRIT",
+  "rarity": "EPIC",
+  "description": "Choose a minion. Summon seven 2/1 Bats to attack it.",
+  "targetSelection": "ENEMY_MINIONS",
+  "spell": {
+    "class": "MetaSpell",
+    "spells": [
+      {
+        "class": "SummonSpell",
+        "cards": [
+          "token_baty",
+          "token_baty",
+          "token_baty",
+          "token_baty",
+          "token_baty",
+          "token_baty",
+          "token_baty"
+        ],
+        "spell": {
+          "class": "AddAttributeSpell",
+          "target": "OUTPUT",
+          "attribute": "RESERVED_BOOLEAN_3"
+        }
+      },
+      {
+        "class": "ExecuteAsSpell",
```

Comment:
I think I understand now that you want everything to be summoned first before its output spell is called, rather than one at a time. I think we can make that change to `SummonSpell`, i.e.,

 1. Summon all minions.
 2. Then, execute the sub spell on each one at a time.